include/library/spdm_crypt_lib.h

```diff
@@ -917,7 +917,7 @@ bool libspdm_x509_certificate_check_ex(const uint8_t *cert, size_t cert_size,
                                        uint32_t base_asym_algo, uint32_t base_hash_algo,
                                        bool is_requester, uint8_t cert_model);
 /**
- * Certificate Check for SPDM leaf cert when set_cert.
+ * Certificate Check for SPDM leaf cert when set_cert. It is used for SPDM 1.2.
```

Comment:
We had better add some reason for the ex/non-ex version difference.
For example, the `basic constrain` is not checked in 1.2, but will be checked in 1.3.